src/WowzaApiClient.php

```diff
@@ -40,4 +40,28 @@ public function setWowzaConfig($wowzaConfig)
     {
         $this->wowzaConfig = $wowzaConfig;
     }
+
+    /**
+     * @param Config $wowzaConfig
+     * @return Integer
+     */
+    public function checkWowzaConfig(Config $wowzaConfig) {
+        $url = $wowzaConfig->getApiUrl() .
+            '/livesetmetadata' .
+            '?app=' . $wowzaConfig->getApp();
```

Comment:
I would prefer to set the query parameters in a more clean way, by setting them into the $option array in the key 'query', as they did here: http://docs.guzzlephp.org/en/stable/quickstart.html#query-string-parameters.